### PR TITLE
[DOC] Add syntax highlighting to MarkDown code blocks

### PR DESCRIPTION
**Pull request by amomchilov**
Split off from https://github.com/ruby/ruby/pull/12322

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -532,7 +532,7 @@ https://github.com/ruby/lrama/pull/62
 
 ### Runtime configuration for error recovery
 
-Make error recovery function configurable on runtime by two new macros.
+Meke error recovery function configurable on runtime by two new macros.
```
Comment (junk0612) [inline NEWS.md:535]: Is this your intended change?

Comment (ydah) [inline NEWS.md:535]:
```suggestion
Make error recovery function configurable on runtime by two new macros.
```

Comment (amomchilov) [reply NEWS.md]: Negative; I don't know how that got in there 😅 

Fixed!

**Review by ydah — APPROVED** [review]
Thank you!

**Pull request merged**